README.md

```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
 image = ThermalImage(image_path="path/to/image", camera_manufacturer="dji/flir")
 ```
```

Comment:
Can you also change this to `from thermal_base import ThermalImage` now that the __init__ file has been modified??